taker/src/main.rs

```diff
@@ -340,6 +341,13 @@ async fn main() -> Result<()> {
         taker_peer_id: identities.libp2p.public().to_peer_id().to_string(),
     };
 
+    let environment = match option_env!("ITCHYSATS_ENVIRONMENT") {
+        Some(environment) => environment
+            .parse()
+            .expect("String that is not processable to default to unknown environment"),
```

Comment:
I am having a hard time processing this error message, would you mind rephrasing it?. also:  shouldn't this rather fallback to unknown env instead of panicking?

Reply:
It does fallback to `Unknown` on the inside. `FromString` is implemented infallible. I an change it to fallback here as well, I don't mind the redundancy.